orbit/pkg/update/runner.go

```diff
@@ -232,6 +233,13 @@ func (r *Runner) needsOrbitSymlinkUpdate() (bool, error) {
 		if errors.Is(err, os.ErrNotExist) {
 			return true, nil
 		}
+
+		if platform.IsInvalidReparsePoint(err) {
+			// On Windows, the symlink may be a file instead of a symlink.
+			// let's handle this case by forcing the update to happen
```

Comment:
This will force an auto-update to the same binary only once right?

Reply:
yes, that's correct. The symlink will be re-generated and next time file won't pass this check